src/problem_bank_scripts/problem_bank_scripts.py

```diff
@@ -470,6 +471,15 @@ def write_info_json(output_path, parsed_question):
         }
     )
 
+    if "workspaceOptions" in info_json: # validate workspaceOptions contains the required keys if it exists
+        image = "image" in info_json["workspaceOptions"]
+        port = "port" in info_json["workspaceOptions"]
+        home = "home" in info_json["workspaceOptions"]
+        if not (image and port and home):
+            raise SyntaxError("workspaceOptions must contain image, port, and home keys")
+        if not isinstance(info_json["workspaceOptions"]["port"], int):
+            raise TypeError(f"workspaceOptions.port must be an integer, got {type(info_json['workspaceOptions']['port'])!r} instead")
+
```

Comment:
I'm not sure if this is really needed, but since workspaces can be a bit picky, I thought adding in a bit of validation for this might not be a bad idea

Let me know if you would like me to create some tests for this